README.md

```diff
@@ -10,24 +10,26 @@
 
 1. [Install Kubecost](https://docs.kubecost.com/install)
 
-> This software requires that you have a running deployment of [Kubecost](https://kubecost.com/) in your cluster. 
+    This software requires that you have a running deployment of [Kubecost](https://kubecost.com/) in your cluster. 
 
+2. Install kubectl cost
 
-#### Linux/MacOS
+    #### Linux/MacOS
 
-``` sh
-os=$(uname | tr '[:upper:]' '[:lower:]') && \
-arch=$(uname -m | tr '[:upper:]' '[:lower:]' | sed -e s/x86_64/amd64/) && \
-curl -s -L https://github.com/kubecost/kubectl-cost/releases/latest/download/kubectl-cost-$os-$arch.tar.gz | tar xz -C /tmp && \
-sudo mv /tmp/kubectl-cost /usr/local/bin/kubectl-cost
-```
+    ``` sh
+    os=$(uname | tr '[:upper:]' '[:lower:]') && \
+    arch=$(uname -m | tr '[:upper:]' '[:lower:]' | sed -e s/x86_64/amd64/) && \
+    curl -s -L https://github.com/kubecost/kubectl-cost/releases/latest/download/kubectl-cost-$os-$arch.tar.gz | tar xz -C /tmp && \
+    sudo mv /tmp/kubectl-cost /usr/local/bin/kubectl-cost
+    ```
 
+    #### Latest Release
 
-#### Latest Release
+    If you prefer to download from GitHub, or are on Windows, go the the [releases](https://github.com/kubecost/kubectl-cost/releases) and download the appropriate     binary for your system. Rename it to `kubectl-cost` and put it in your `PATH`.
 
-If you prefer to download from GitHub, or are on Windows, go the the [releases](https://github.com/kubecost/kubectl-cost/releases) and download the appropriate binary for your system. Rename it to `kubectl-cost` and put it in your `PATH`.
+    As long as the binary is still named `kubectl-cost` and is somewhere in your `PATH`, it will be usable. This is because `kubectl` automatically finds plugins by looking for executables prefixed with `kubecost-` in your `PATH`.
```

Comment:
```suggestion
    As long as the binary is still named `kubectl-cost` and is somewhere in your `PATH`, it will be usable. This is because `kubectl` automatically finds plugins by looking for executables prefixed with `kubectl-` in your `PATH`.
```

Looks like I found a typo of my own, too.